src/main.rs

```diff
@@ -81,10 +81,10 @@ fn main() {
     let batch_size: u16 = AVERAGE_BATCH_SIZE;
 
     // Added by wasuaje - 01/26/2024:
-    // exclude_ports  is an exclusion port list
+    // exclude_ports is an exclusion port list
     //
     // Added by brendanglancy - 5/19/2024:
-    // udp is an option to do a udp scan
+    // udp is an option to do an udp scan
```

Comment:
All the other changes make sense but my take is that this is already correct: if this is pronounced "yoo-dee-pee", it should use "a" as an article.